downstream/assemblies/platform/assembly-disconnected-installation.adoc

```diff
@@ -6,6 +6,9 @@ ifdef::context[:parent-context: {context}]
 [id="disconnected-installation"]
 = Disconnected installation
 
+[role="_abstract"]
+{PlatformName} can be installed without an active internet connection.
+
```

Comment:
It might be good to help an end user understand why this is important. Is this an added benefit for AAP (the fact that you can still install AAP without being connected to the internet)? Just a thought.

Also, it might be good to include a high-level (possibly bullet) list of what the main headings are (maybe add links to each of the procedures included in this process).

Reply:
I added a little more to this initial abstract, but I think once we restructure everything this assembly will no longer be the first assembly in the chapter, so I may save writing the introductory abstract until after that.

Reply:
Sounds good.